### ISSUE TITLE
Accessibility issues in paginator component

### ISSUE THREAD

**Issue by Faiz-exe** [primefaces/primereact#7736]
### Describe the bug

Observed the following issues while running axeDevtools on Prime React paginator component

1.All page contents should be contained by land marks

`<div class="p-hidden-accessible" data-pc-section="hiddenselectedmessage"><input type="text" readonly="" aria-haspopup="listbox" tabindex="0" aria-label="Jump to Page Dropdown" data-pc-section="input" value="10"></div>`

`<div class="p-hidden-accessible p-dropdown-hidden-select" data-pc-section="hiddenselectedmessage"><select tabindex="-1" data-pc-section="select"><option value="10" data-pc-section="option" selected="">10</option></select></div>`

`<span class="p-dropdown-label p-inputtext" tabindex="-1" data-pc-section="input">10</span>`

![Image](https://github.com/user-attachments/assets/6c52b096-2789-4350-b065-d1387e7e86e2)

2.ARIA input fields must have an accessible name 

`<ul class="p-dropdown-items" role="listbox" data-pc-section="list">`

![Image](https://github.com/user-attachments/assets/d878eea9-42b1-443a-8008-67d07c0d672a)

3.Ensure select element has an accessible name

`<select tabindex="-1" data-pc-section="select"><option value="10" data-pc-section="option" selected="">10</option></select>`

![Image](https://github.com/user-attachments/assets/45b996f1-495a-4c1d-964f-1a69f3719d11)

To reproduce this , go to prime react documentation page ,https://primereact.org/paginator/
open the drop down , run axeDevtools and observe the issues 

### System Information

```Shell
PrimeReact Version: 10.9.1
Browser: Chrome
Operating System: Windows
```

### Steps to reproduce the behavior

1. Navigate to the PrimeReact dropdown documentation page. https://primereact.org/paginator/
2. **Open paginator drop down**
3. Run axeDev tools.
4. Observe the issue flagged in the accessibility section of the report.
